cfe_internal/enterprise/CFE_knowledge.cf

```diff
@@ -192,37 +192,45 @@ bundle agent cfe_internal_setup_knowledge
         comment => "The agent will complain if any other users (group or other)
                     have write access to the modules directory.";
 
+}
+
+bundle agent cfe_internal_permissions
+# @brief Specific expectations for permissions and ownership of CFEngine with respect to the Enterprise Edition
+{
+
+  files:
+
+    !(policy_server|am_policy_hub)::
+      "$(sys.statedir)/."
+        perms => mog("0600", "root", "root"),
+        # Important to recurse across file system boundaries, as databases and or state are commonly on different filesystems
+        depth_search => recurse_with_base( inf ),
+        file_select => all_but_reporting_tmp;
```

Comment:
Fails in jenkins, this body does not exist, in 3.10:

```
3.10.6a.c2357822/tests/acceptance/workdir/__00_basics_def_json_control_common_bundlesequence_end_cf/inputs/cfe_internal/enterprise/CFE_knowledge.cf:208:0: error: Undefined body all_but_reporting_tmp with type file_select
```

Reply:
Hum that was a body I was just temporarily testing with, it's supposed to be `all`